### PR TITLE
Add two new toTask method to PlayParSeq that accept Executor param

### PR DESCRIPTION
**Pull request by chenfanggm**
As title.

### CODE REVIEW

File: src/core/java/app/com/linkedin/playparseq/j/PlayParSeq.java [hunk 1]
```diff
@@ -38,6 +39,17 @@
...
+  /**
+   * The method toTask converts a {@code Callable<CompletionStage<T>>} to a ParSeq {@code Task<T>}.
+   *
+   * @param name The String which describes the Task and shows up in a trace
+   * @param executionContext The Executor of the async CompletionStage
+   * @param f The Callable which returns a CompletionStage
+   * @param <T> The type parameter of the CompletionStage and the ParSeq Task
+   * @return The ParSeq Task
+   */
+  <T> Task<T> toTask(final String name, final HttpExecutionContext executionContext, final Callable<CompletionStage<T>> f);
```
Comment (FranklinYinanDing) [inline src/core/java/app/com/linkedin/playparseq/j/PlayParSeq.java]: To match CompletionStage's API style, I suggest moving `HttpExecutionContext` to the last parameter. Same elsewhere.

File: src/core/java/app/com/linkedin/playparseq/j/PlayParSeqImpl.java [hunk 2]
```diff
@@ -79,6 +80,25 @@ public PlayParSeqImpl(final Engine engine, final ParSeqTaskStore parSeqTaskStore
...
+      f.call().whenCompleteAsync((result, exception) -> {
+        if (exception != null) {
+          promise.fail(exception);
+        } else {
+          promise.done(result);
+        }
+      }, executionContext.current());
+      return promise;
+    });
+  }
```
Comment (FranklinYinanDing) [inline src/core/java/app/com/linkedin/playparseq/j/PlayParSeqImpl.java]: Could you consolidate this new method with the above? So we don't rely on copying code.

**Review by FranklinYinanDing — CHANGES_REQUESTED** [review]

File: src/core/java/app/com/linkedin/playparseq/j/PlayParSeq.java (hunk 1 above)
Comment (chenfanggm) [reply src/core/java/app/com/linkedin/playparseq/j/PlayParSeq.java]: Got it. Will do.

File: src/core/java/app/com/linkedin/playparseq/j/PlayParSeqImpl.java (hunk 2 above)
Comment (chenfanggm) [reply src/core/java/app/com/linkedin/playparseq/j/PlayParSeqImpl.java]: Great call.

**Pull request merged**